File: library/src/main/java/com/adedom/library/MyLibrary.java
package com.adedom.library;

import android.content.Context;
import android.widget.EditText;
import android.widget.Toast;

public class MyLibrary {

    public static void completed(Context context) {
        Toast.makeText(context, R.string.completed, Toast.LENGTH_SHORT).show();
    }

    public static void failed(Context context) {
        Toast.makeText(context, R.string.failed, Toast.LENGTH_LONG).show();
    }

    public static Utility with(Context context) {
        return new Utility(context);
    }

    public static Boolean isEmpty(EditText editText, String error) {
        if (editText.getText().toString().trim().isEmpty()) {
            editText.requestFocus();
            editText.setError(error);
            return true;
        }
        return false;
    }
}

File: utility/src/main/java/com/adedom/utility/MyLibrary.kt
package com.adedom.utility

import android.content.Context
import android.widget.EditText
import android.widget.Toast

class MyLibrary {

    companion object{
        fun completed(context: Context) {
            Toast.makeText(context, R.string.completed, Toast.LENGTH_SHORT).show()
        }

        fun failed(context: Context) {
            Toast.makeText(context, R.string.failed, Toast.LENGTH_LONG).show()
        }

        fun with(context: Context): Utility {
            return Utility(context)
        }

        fun isEmpty(editText: EditText, error: String): Boolean {
            if (editText.text.toString().trim().isEmpty()) {
                editText.requestFocus()
                editText.error = error
                return true
            }
            return false
        }
    }
}